tempesta_fw/cache.c

```diff
@@ -1144,11 +1144,12 @@ cache_req_process_node(TfwHttpReq *req, unsigned long key,
 	TfwHttpResp *resp = NULL;
 	TDB *db = node_db();
 	TdbIter iter;
+	time_t lifetime = 0;
```

Comment:
`lifetime` is never used uninitialized, so there is no sense to initialize it with zero.